OJP/OJP_JourneySupport.xsd

```diff
@@ -93,6 +101,11 @@
 				</xs:annotation>
 			</xs:element>
 			<xs:element ref="OperatorRef" minOccurs="0"/>
+			<xs:element name="additionalOperators" type="OperatorRefList" minOccurs="0">
```

Comment:
The meaning in NeTEx is different. As in multiple operators execute this leg.

Reply:
@skinkie  Should I alter the name to what? or make it directly multiple OperatorRef?

Reply:
I wonder why NeTEx choose the additionalOperators in the first place...

@Aurige ?

Reply:
simpflied to OperatorRef "unbounded"